draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -160,9 +162,11 @@
 
        Charset Dependent: no
 
+       Default Value: empty string
+
        Syntax:
 
-           conn-value = 1*256alphanum
+           conn-value = 0*256alphanum
 

```

Comment:
Q1: I don't the value shall be a "string", because that normally means quoted are needed. I suggest to say "empty value".

Q2: We should use something else than "conn-value". "id-value", or something.